pkg/services/alerting/notifier.go

```diff
@@ -127,7 +127,7 @@ func (n *notificationService) uploadImage(context *EvalContext) (err error) {
 	renderOpts := rendering.Opts{
 		Width:           1000,
 		Height:          500,
-		Timeout:         time.Duration(setting.AlertingEvaluationTimeout.Seconds() * 0.9),
+		Timeout:         time.Duration(setting.AlertingEvaluationTimeout.Seconds()*0.9) * time.Second,
```

Comment:
If `setting.AlertingEvaluationTimeout` is a `time.Duration` I think 
`setting.AlertingEvaluationTimeout * 0.9` will be enough

Reply:
Hmm right even lint is complaining about that.

Reply:
Hmm ok the lint was not about this one. I am not sure if this can be simplified. `setting.AlertingEvaluationTimeout * 0.9` complains that `0.9` cannot be converted to int64 which makes sense I assume.

Reply:
Same complaint with `setting.AlertingEvaluationTimeout.Seconds() * 0.9 * time.Second`

Reply:
Hmm question is if need the 0.9 there at all now when we have one configuration each for evaluation and notification. Or do we still have an overall timeout which is the sum of both? @bergquist maybe can shed some light on this.

Reply:
Your right. We can get rid of it

Reply:
```suggestion
		Timeout:         setting.AlertingEvaluationTimeout,
```

Reply:
Once this is done this is ready to be merged.